src/internal/packager/images/progress_test.go

```diff
@@ -35,7 +35,7 @@ func TestTracker_ReportingCycle(t *testing.T) {
 		lastTotalBytes.Store(totalBytes)
 	}
 
-	testInterval := 10 * time.Millisecond
+	testInterval := 20 * time.Millisecond
```

Comment:
This flaked on windows. If it happens again we may just rework this test, but I figure I'll try this to start